code/game/objects/items/stacks/sheets/glass.dm

```diff
@@ -109,6 +128,26 @@ GLOBAL_LIST_INIT(reinforced_glass_recipes, list ( \
 /obj/item/stack/sheet/rglass/Initialize(mapload, new_amount, merge = TRUE)
 	recipes = GLOB.reinforced_glass_recipes
 	..()
+	
+GLOBAL_LIST_INIT(prglass_recipes, list ( \
+	new/datum/stack_recipe("directional reinforced window", /obj/structure/window/plasma/reinforced/unanchored, time = 0, on_floor = TRUE, window_checks = TRUE), \
+	new/datum/stack_recipe("fulltile reinforced window", /obj/structure/window/plasma/reinforced/fulltile/unanchored, 2, time = 0, on_floor = TRUE, window_checks = TRUE) \
+))
+
+/obj/item/stack/sheet/rglass/plasma
+	name = "reinforced plasma glass"
+	desc = "Plasma glass which seems to have rods or something stuck in them."

```

Comment:
This needs a better desc. Perhaps "A plasma-glass alloy fortified with metal rods.", or such?